src/document/DocumentAnnotator.ts

```diff
@@ -57,6 +58,18 @@ export default class DocumentAnnotator extends BaseAnnotator {
         return this.annotatedEl ? Array.from(this.annotatedEl.querySelectorAll('.page')) : [];
     }
 
+    toggleAnnotationMode(mode: Mode): void {
```

Comment:
Should we manage this via an event listener on the store, instead? If the mode is set any other way besides this method, the class list will be stale.